api/envoy/config/api_listener/v2/api_listener.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package envoy.config.api_listener.v2;
```

Comment:
Do you want to make this `v2alpha` until you have verified this works?

Reply:
I don't think that would actually buy us anything, because we need to use it from the v2 LDS API.  But I really don't expect any significant changes here, especially since we're using `Any` for now; if we do need something else, we can just change what message we're using in the `Any`.